cdc/sink/mysql.go

```diff
@@ -345,6 +353,34 @@ func (s *mysqlSink) Close() error {
 	return nil
 }
 
+func (s *mysqlSink) PrintStatus(ctx context.Context) error {
+	changefeedID := util.ChangefeedIDFromCtx(ctx)
```

Comment:
if the caller of sink is owner, could we get changefeedId from context?

how about store a value in context, in order to get the role of caller?

Reply:
by the way, maybe we can storage `changefeedId` in `mysqlSink` struct to avoid query changedfeedId  continually

Reply:
Add more options in `sink opt`, as `changefeedID` and `captureID` are used quite often. The `role` is not put into sink now, as we only call `PrintStatus` of the sink in processor. We can add the role information if we really need it.